lawlibrary/urls.py

```diff
@@ -1,21 +1,8 @@
-from django.conf import settings
-from django.conf.urls.static import static
 from django.urls import include, path
 
 from lawlibrary import views
-from peachjam.urls import common_url_patterns
 
-urlpatterns = common_url_patterns + [
-    path("", views.HomePageView.as_view(), name="home_page")
+urlpatterns = [
+    path("", views.HomePageView.as_view(), name="home_page"),
+    path("", include("peachjam.urls")),
```

Comment:
in theory this should include "lawlibrary.urls" which in turn includes "peachjam.urls", so that we keep the full "inheritance" hierarchy.

Reply:
done